hasjob/static/sass/_header.sass

```diff
@@ -1,6 +1,6 @@
 body
   font-family: $font-default
-  background: #e8e7e6 image-url("canvas.jpg")
+  background: #e8e7e6 url("~/static/img/canvas.jpg")
```

Comment:
We should move SASS files into `assets` as well, which means this path will need to be updated (image files stay in `static`).

Reply:
Fixed